bagel/cli.py

```diff
@@ -21,16 +21,27 @@
 
 CUSTOM_SESSION_LABEL = "ses-unnamed"
 
-bagel = typer.Typer(
-    help="""
-    A command-line tool for creating valid, subject-level instances of the Neurobagel graph data model.\n
-    The 'pheno' command must always be run first to generate the input .jsonld file required for the 'bids' command.
+OPTION_GROUP_NAMES = {
+    "general": "Troubleshooting",
```

Comment:
Why do we use "general" as the key for Troubleshooting here? Wouldn't 
```suggestion
    "troubleshooting": "Troubleshooting",
```
make more sense?